pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitLab , which, go-md2man, coreutils, substituteAll }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.4";
+  pname = "brillo";
+  src = fetchFromGitLab {
+    owner= "cameronnemo";
+    repo= "brillo";
+    rev= "v${version}";
+    sha256 = "0sadsqjxwvmq2jrvq8skvix3jg936k1qsv1x2hw8v9w6craghm99
```

Comment:
Please correct the hash.

All my previous request for changes look resolved to me :+1:

Reply:
Sorry about that, must have crept in while rebasing